cli/format/format_extended.go

```diff
@@ -132,14 +132,7 @@ func extendedFormatAnnotations(labels models.LabelSet) string {
 func extendedFormatMatchers(matchers models.Matchers) string {
 	output := []string{}
 	for _, matcher := range matchers {
-		output = append(output, extendedFormatMatcher(*matcher))
+		output = append(output, simpleFormatMatcher(*matcher))
```

Comment:
Similar to below, perhaps we can re-use the `label.Matchers.String()` method here?

What I suggested below for line 97 could perhaps be put into a helper function. Then we can easily turn `models.Matchers` into `labels.Matchers` and call `String()` on it.

Reply:
I just see that cli/utils.go already has `TypeMatcher` and `TypeMatchers` for the conversion in the other direction. So let's just create the inverse functions there as well, which will make the formatting very easy.

Reply:
I will make the reverse function in the `cli/format` package (because `cli/format` can not import `cli`) and will NOT export it. Otherwise compilation and error handling should happen. We only want to represent it as string and compiling (if `MatchRegexp` or `MatchNotRegexp`) is useless. But if we export this function, somebody somewhere else might start using it for something completely different thus ending up with half-working matcher value.